gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/AbstractGremlinTest.java

```diff
@@ -173,7 +189,7 @@ public Object convertToVertexId(final Graph graph, final String vertexName) {
 
     public Vertex convertToVertex(final Graph graph, final String vertexName) {
         // all test graphs have "name" as a unique id which makes it easy to hardcode this...works for now
-        return graph.traversal().V().has("name", vertexName).next();
+        return graphProvider.traversal(graph).V().has("name", vertexName).toList().get(0);
```

Comment:
Did you learn something here? Is there a need for the test suite to prefer `graphProvider.traversal(graph)` over simply `graph.traversal()`? I just see that you made such changes here, but in other places you didn't make such changes:

https://github.com/apache/tinkerpop/pull/1118/files#diff-d201caccfbd94491ed409d1f26ec5b62R149

Reply:
The purpose of this change was to consume the iterator completely so that the underlying resources are closed and hence, the `toList().get(0)` is the significant part in the code change above. The traversal can be obtained in any manner. I have reverted the first part to use graph.traveral() itself to minimize changes.

Reply:
i just started to merge this and i think that change might have been an artifact from your rebase at some point as i see that style in `tp34`. sorry about making you change that.  i will probably go back and add that change to `tp33` after this merge is done.